packages/item/src/vaadin-item-styles.js

```diff
@@ -19,7 +19,7 @@ export const itemStyles = css`
       padding: var(--vaadin-item-padding, var(--_vaadin-padding-container));
     }
 
-    :host(:is(:focus-visible, vaadin-combo-box-item[focused])) {
+    :host(:is(:focus-visible, vaadin-combo-box-item[focused], vaadin-time-picker-item[focused])) {
```

Comment:
We will also need to add `vaadin-multi-select-combo-box-item` here later.

Reply:
So perhaps the behavior of Select should be changed a bit, so that it focuses the overlay first, instead of the first item, when you open the overlay with the mouse/pointer device.

Reply:
Isn't that how Menu Bar and Context Menu work? Personally I would expect the same behavior in these components. The native `<select>` also works like this. Though in macOS the mouse moves the keyboard focus (aka "focus follows mouse" within the overlay) and when you open the select the current selected item is under the mouse so keyboard focus is there. But when you move the mouse outside the overlay, then keyboard focus is apparently moved to the overlay, and pressing arrow down focuses the first item.

Reply:
I changed this back to the simple `:host([focused])` version.